useful_resources/computer_setup/python3_setup.md

```diff
@@ -0,0 +1,57 @@
+# Installing Python 3
+
+
+## Why Python 3?
+
+As you may have heard, starting [January 1, 2020][countdown] [Python 2.x is no
+longer officially supported][eol_article].
+
+Additionally, when using [Parsons][parsons], we use Python 3. Therefore, to make
+collaboration, debugging and maintenance easier, we encourage that you install and
+use Python 3 when writing scripts for TMC.
```

Comment:
Perhaps rephrase along the lines of: _Additionally, Parsons is only compatible with Python 3. So, in order to use parsons, you need to have Python 3 installed._